package.json

```diff
@@ -31,11 +32,34 @@
     "type": "git",
     "url": "https://github.com/mswjs/interceptors"
   },
+  "exports": {
+    ".": {
+      "require": "./lib/index.js",
+      "import": "./lib/index.mjs",
+      "types": "./lib/index.d.ts"
+    },
+    "./lib/interceptors/ClientRequest": {
+      "require": "./lib/interceptors/ClientRequest/index.js",
+      "import": "./lib/interceptors/ClientRequest/index.mjs",
+      "types": "./lib/interceptors/ClientRequest/index.d.ts"
+    },
+    "./lib/interceptors/fetch": {
```

Comment:
Should we drop `./lib/interceptors/` from the exports paths? So that you could import interceptors like so:

```js
import X from '@mswjsw/interceptors/ClientRequest'
```

The more we work on this, the more I think it needs to be a separate change. I wouldn't want to ask you to restructure all of your work, so if you feel it's already under substantial progress, we can ship this together with the `remix-run` change. If not, I think we're safer releasing the `exports` changes separately.

Reply:
I've moved it to a separate PR. Let's tackle that once we get the new bundling merged since I'm not sure if it depends on this work or not.
I've opened a draft PR on my fork just to keep a record of the work: https://github.com/cdimitroulas/interceptors/pull/1

Reply:
I don't think these changes are dependent. Thanks for moving it away 👍